magenta/models/sketch_rnn/utils.py

```diff
@@ -195,11 +208,15 @@ def __init__(self,
                limit=1000):
     self.batch_size = batch_size  # minibatch size
     self.max_seq_length = max_seq_length  # N_max in sketch-rnn paper
-    self.scale_factor = scale_factor  # divide data by this factor
+    self.scale_factor = scale_factor  # divide offsets by this factor
     self.random_scale_factor = random_scale_factor  # data augmentation method
-    self.limit = limit  # removes large gaps in the data
+    # Removes large gaps in the data. x and y offsets are clamped to have
+    # absolute value no greater than this limit.
```

Comment:
I don't totally understand this part. I would think "large gaps" means large distances between a "pen up" and "pen down" state, but from my reading of `preprocess()`, it seems like this actually clamps all offsets. Meaning it would also shrink line segments having an x or y component greater than `limit`. Is that intentional? Or maybe in practice this doesn't happen because some earlier preprocessing means there are no line segments with offsets greater than 1000 or other typical `limit` values?